ibis/tests/all/test_geospatial.py

```diff
@@ -1,102 +1,146 @@
 """ Tests for geo spatial data types"""
-from inspect import isfunction
+import sys
 
 import numpy as np
 import pytest
 from numpy import testing
 from pytest import param
 
 import ibis
-from ibis.tests.backends import MapD
+from ibis.tests.backends import MapD, PostgreSQL
+
+FULL_GEO_SUPPORTED = False
+try:
+    # supported just for python >= 3.6
+    if sys.version_info >= (3, 6):
```

Comment:
Can you use the standard `pytest` skipping machinery here? E.g., `pytestmark = pytest.mark.skipif(sys.version_info < (3, 6))` and `geopandas = pytest.importorskip('geopandas')`?